openapi/p4pa-classification.openapi.yaml

```diff
@@ -496,3 +524,118 @@ components:
           type: string
         printed:
           type: boolean
+    PersonDTO:
```

Comment:
if Person already exists, why don't you typeMap it instead of defining again it?